src/test/java/it/gov/pagopa/afm/calculator/service/CalculatorServiceTest.java

```diff
@@ -362,27 +369,6 @@ void calculate_paymentType_Null() throws IOException, JSONException {
 
   @Test
   @Order(13)
-  void calculate_BinNull() throws IOException, JSONException {
```

Comment:
Why was this test cancelled?

Reply:
because for cleaning I merged some similar tests into one parameterized one
```
@ParameterizedTest
  @CsvSource({
      "requests/getFees.json, responses/getFees.json",
      "requests/getFeesBinNull.json, responses/getFeesBinNull.json",
      "requests/getFeesPspList.json, responses/getFees.json"
  })
  @Order(1)
  void calculate(String input, String output) throws IOException, JSONException
```

Reply:
perfect